languageserver/integration/commands.go

```diff
@@ -86,42 +97,36 @@ func (i *FlowIntegration) commands() []server.Command {
 //   * the arguments, encoded as JSON-CDC
 func (i *FlowIntegration) submitTransaction(conn protocol.Conn, args ...interface{}) (interface{}, error) {
 
-	err := server.CheckCommandArgumentCount(args, 2)
+	err := server.CheckCommandArgumentCount(args, 3)
 	if err != nil {
 		return nil, err
 	}
 
-	uri, ok := args[0].(string)
-	if !ok {
-		return nil, fmt.Errorf("invalid URI argument: %#+v", args[0])
-	}
-
-	doc, ok := i.server.GetDocument(protocol.DocumentUri(uri))
+	_, ok := args[1].([]interface{})
 	if !ok {
-		return nil, fmt.Errorf("could not find document for URI %s", uri)
+		return nil, fmt.Errorf("invalid transaction arguments: %#+v", args[1])
 	}
 
 	rawTransactionArguments, ok := args[1].([]interface{})
 	if !ok {
-		return nil, fmt.Errorf("invalid transaction arguments: %#+v", args[1])
+		return nil, fmt.Errorf("invalid script arguments: %#+v", args[1])
 	}
 
-	script := []byte(doc.Text)
-
-	tx := flow.NewTransaction().
-		SetScript(script).
-		AddAuthorizer(i.activeAddress)
-
+	var transactionArguments []string
 	for i, rawTransactionArgument := range rawTransactionArguments {
-		transactionArgumentString, ok := rawTransactionArgument.(string)
+		stringArgument, ok := rawTransactionArgument.(string)
 		if !ok {
 			return nil, fmt.Errorf("invalid transaction argument at index %d: %#+v", i, rawTransactionArgument)
 		}
 
-		tx.AddRawArgument([]byte(transactionArgumentString))
+		transactionArguments = append(transactionArguments, strings.TrimSuffix(stringArgument, "\n"))
 	}
 
-	_, err = i.sendTransactionHelper(conn, i.activeAddress, tx)
+	// Pass arguments back to extension
+	codeArguments := fmt.Sprintf("%s", strings.Join(transactionArguments, ","))
+	signers := args[2].([]interface{})
```

Comment:
Please validate the input and handle it gracefully:

```suggestion
	signers, ok := args[2].([]interface{})
	if !ok {
		return nil, fmt.Errorf("invalid signers argument: %#+v", args[2])
	}
```

Reply:
I added proposed changes, though this code currently is not used, but will be refactored to use shared CLI library...